docs/openrpc.json

```diff
@@ -1251,6 +1252,19 @@
                             "s": {
                                 "title": "s",
                                 "$ref": "#/components/schemas/uint"
+                            },
+                            "v": {
+                                "title": "v",
+                                "description": "For backwards compatibility",
```

Comment:
I suggest using the same description provided by the official Ethereum specs:

```text
For backwards compatibility, `v` is optionally provided as an alternative to `yParity`. This field is DEPRECATED and all use of it should migrate to `yParity`.
```

Reply:
Good suggestion, Thanks!!

✅  Done